### PR TITLE
fzf: select from git tracked files

### PR DESCRIPTION
**Pull request by ryanolsonx**
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

**Comment by zdroid** [comment]
Thanks for your recent contributions, keep up the good work!

File: pages/common/fzf.md [hunk 1]
```diff
@@ -7,6 +7,10 @@
 
 `find {{path/to/search}} -type f | fzf`
 
+- Start finder on all files tracked by git:
```
Comment (agnivade) [inline pages/common/fzf.md]: I don't see the usefulness of this command. This does not show a new feature of fzf. And looks very similar to the first and third examples.

Comment (ryanolsonx) [reply pages/common/fzf.md]: It's very useful.  While similar to other commands, many users might be unaware that they can search within their git tracked files.

Comment (agnivade) [reply pages/common/fzf.md]: I mean can't they make that conclusion from the 1st and 3rd command ? It seems natural to me to pipe the output of any list of files to fzf and search within it. How does this specific command help it further ?

Comment (ryanolsonx) [reply pages/common/fzf.md]: Fair enough.

Comment (agnivade) [reply pages/common/fzf.md]: Thanks @ryanolsonx. Your contributions are really appreciated though ! Please don't let this discourage you from sending more changes.

Comment (ryanolsonx) [reply pages/common/fzf.md]: Thanks

**Pull request closed without merging**